qiskit_experiments/framework/composite/batch_experiment.py

```diff
@@ -89,6 +93,19 @@ def __init__(
             experiments, qubits, backend=backend, analysis=analysis, flatten_results=flatten_results
         )
 
+    @property
```

Comment:
These property and setter already exist in `BaseExperiment`, so there is no reason to copy them here.

Note that all the comments for the `BatchExperiment` apply also to the `ParallelExperiment`.

Reply:
fixed

Reply:
What do you mean by "fixed"? Did you commit the fix? Because the property and setter are still here.